proto/osmosis/poolmanager/v1beta1/genesis.proto

```diff
@@ -17,6 +17,21 @@ message Params {
     (gogoproto.moretags) = "yaml:\"pool_creation_fee\"",
     (gogoproto.nullable) = false
   ];
+  // taker_fee_params is the container of taker fee parameters.
+  TakerFeeParams taker_fee_params = 2 [
+    (gogoproto.moretags) = "yaml:\"taker_fee_params\"",
+    (gogoproto.nullable) = false
+  ];
+  // authorized_quote_denoms is a list of quote denoms that can be used as
+  // token1 when creating a concentrated pool. We limit the quote assets to a
+  // small set for the purposes of having convinient price increments stemming
+  // from tick to price conversion. These increments are in a human readable
+  // magnitude only for token1 as a quote. For limit orders in the future, this
+  // will be a desirable property in terms of UX as to allow users to set limit
+  // orders at prices in terms of token1 (quote asset) that are easy to reason
+  // about.
+  repeated string authorized_quote_denoms = 3
+      [ (gogoproto.moretags) = "yaml:\"authorized_quote_denoms\"" ];
```

Comment:
Note to reviewer:  As mentioned earlier, this was moved from CL params to here. It was not removed from CL params yet since we migrate in this upgrade, and then next upgrade we can completely remove from CL.

Reply:
I had massive issues importing CL keeper into pool manager with cyclical imports, but if you can figure it out I'm good with switching to this

Reply:
OK. Let's not block on this. Could you make an issue to remove the param from CL after the upgrade please?

By remove, I mean marking it as reserved in the proto in the next major upgrade